src/Component/Calculator.js

```diff
@@ -1,47 +1,70 @@
+            <button className="p-5 orange" onClick={() => { this.eventHandler(calculate(this.state, '÷')); }}>÷
+</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5">7</button>
-            <button className="p-5">8</button>
-            <button className="p-5">9</button>
-            <button className="p-5 orange">x</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '7')); }} >7</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '8')); }}>8</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '9')); }}>9</button>
+            <button className="p-5 orange" onClick={() => { this.eventHandler(calculate(this.state, 'x')); }}>x</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5">4</button>
-            <button className="p-5">5</button>
-            <button className="p-5">6</button>
-            <button className="p-5 orange">-</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '4')); }}>4</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '5')); }}>5</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '6')); }}>6</button>
+            <button className="p-5 orange" onClick={() => { this.eventHandler(calculate(this.state, '-')); }}>-</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5">1</button>
-            <button className="p-5">2</button>
-            <button className="p-5">3</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '1')); }}>1</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '2')); }}>2</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '3')); }}>3</button>
             <button className="p-5 orange">+</button>
           </div>
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5 button-2">0</button>
-            <button className="p-5">.</button>
-            <button className="p-5 orange">=</button>
+            <button className="p-5 button-2" onClick={() => { this.eventHandler(calculate(this.state, '0')); }}>0</button>
+            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '.')); }}>.</button>
+            <button className="p-5 orange" onClick={() => { this.eventHandler(calculate(this.state, '=')); }}>=</button>
           </div>
         </div>
```

Comment:
- [ ] While you have done a great job with this project but I would like to suggest you work on the functionality of the app. I cannot solve all the problems correctly as we can do on a normal calculator do.

Reply:
Hello, Please what are the functionalities? Which of the problem? Because everything works fine here

Reply:
Hi there, 
Thank you for reaching out. 
When I run the application, for a few problems handle click doesn't work. And for those it works such as multiplication, it would show X with the answer.

Reply:
Well, I can't replicate this issue

Reply:
- [ ]  This issue is not resolved. Kindly see the review.